trino/trino.go

```diff
@@ -909,6 +921,14 @@ func (st *driverStmt) exec(ctx context.Context, args []driver.NamedValue) (*stmt
 				continue
 			}
 
+			if arg.Name == accessTokenConfig {
+				if st.conn.forwardAuthorizationHeader {
+					token := arg.Value.(string)
+					hs.Add(authorizationHeader, getAuthorization(token))
+				}
+				continue
```

Comment:
I first check that this parameter is the access token (`if arg.Name == accessTokenConfig`), then check if the `ForwardAuthorizationHeader` is true. If not, I think we still have to ignore it because it introduces some unwanted behavior, like an access token, into the request body.

Reply:
What if someone uses this as an actual query parameter?

Reply:
Ok, I see the point.I'm changing that

Reply:
Sorry for coming back on this but I'm not familiar with the `useExplicitePrepare` parameter and it looks like the token falls in this case when I don't ignore the `accessToken` arg.
Then I get this in request body:

```
EXECUTE _trino_go USING '<accessToken>'
```
and `x-trino-prepared-statement
_trino_go=SELECT+1` supplementary request header.

Is this a wanted behavior or is it a side effect?

Reply:
It's expected, you'd have to have a query with at least one parameter, maybe `SELECT ?` instead of `SELECT 1`.

Reply:
I was too focused on integrating `trino-go-client` for `grafana-trino` and overlooked compatibility with other tools.
I’m addressing this now and will squash the commits.